website/source/docs/providers/tls/r/locally_signed_cert.html.md

```diff
@@ -8,7 +8,7 @@ description: |-
 
 # tls\_locally\_signed\_cert
 
-Generates a TLS ceritifcate using a *Certificate Signing Request* (CSR) and
+Generates a TLS certificate using a *Certificate Signing Request* (CSR) and
```

Comment:
This is a good change, but not related to this PR. It'd be awesome if you could submit it as a separate PR, which we can then merge very quickly. :grinning: